app/src/main/java/ca/dal/cs/athletemonitor/athletemonitor/UserLocation.java

```diff
@@ -0,0 +1,129 @@
+package ca.dal.cs.athletemonitor.athletemonitor;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.util.SparseIntArray;
+
+/**
+ * This class describes a user associated with a location for display
+ * in the MapsActivity.
+ */
+public class UserLocation implements Parcelable {
+
+    public static final SparseIntArray IMAGE_ID_MAP = new SparseIntArray();
+
+    /*static {
+        IMAGE_ID_MAP.put(0, R.drawable.ic_map_bicycle);
+        IMAGE_ID_MAP.put(1, R.drawable.ic_map_face);
+        IMAGE_ID_MAP.put(2, R.drawable.ic_map_film);
+        IMAGE_ID_MAP.put(3, R.drawable.ic_map_key);
+        IMAGE_ID_MAP.put(4, R.drawable.ic_map_monkey);
+        IMAGE_ID_MAP.put(5, R.drawable.ic_map_music);
+        IMAGE_ID_MAP.put(6, R.drawable.ic_map_paw);
+        IMAGE_ID_MAP.put(7, R.drawable.ic_map_pizza);
+        IMAGE_ID_MAP.put(8, R.drawable.ic_map_row);
+        IMAGE_ID_MAP.put(9, R.drawable.ic_map_run);
+        IMAGE_ID_MAP.put(10, R.drawable.ic_map_smiley);
+        IMAGE_ID_MAP.put(11, R.drawable.ic_map_swim);
+        IMAGE_ID_MAP.put(12, R.drawable.ic_map_walk);
+        IMAGE_ID_MAP.put(13, R.drawable.ic_map_weight);
+        IMAGE_ID_MAP.put(14, R.drawable.ic_map_wheelchair);
+    }*/
```

Comment:
Not a big deal, since it'll be picked up in our merge, but this doesn't need to be commented out

Reply:
If they don't comment it out, it'll probably have a compilation error, because the drawable resources won't exist.